src/Domain/Composer/ComposerFacade.php

```diff
@@ -131,6 +131,7 @@ public function createProjectFromPackage(Package $package): void {
     ]);
     $this->runComposer([
       'create-project',
+      '--stability=dev',
```

Comment:
This needs to be conditional. Only use `--stability=dev` if the SUT is a project template.

Otherwise, a project running a stable Drupal 9 job could accidentally install a D10 release of the project template, e.g. 10.0.0-alpha1, the same bug as ORCA-446

Reply:
@danepowell : Actually this function only runs when the SUT _is a_ project template. If the SUT _is not a_ project template ORCA runs a different function. Please check below

_https://github.com/acquia/orca/blob/895edf8c914f5ed8bf6d762cf7e58ba54c3b329d/src/Domain/Fixture/CodebaseCreator.php#L104-L119_